examples/model_selection/plot_successive_halving_iterations.py

```diff
@@ -27,13 +27,13 @@
 
 rng = np.random.RandomState(0)
 
-X, y = datasets.make_classification(n_samples=700, random_state=rng)
+X, y = datasets.make_classification(n_samples=400, n_features=12, random_state=rng)
 
-clf = RandomForestClassifier(n_estimators=20, random_state=rng)
+clf = RandomForestClassifier(n_estimators=15, random_state=rng)
```

Comment:
Can we leave this at `20`? I like how the last iteration shows a difference between the last two candidates:

![Figure_1](https://user-images.githubusercontent.com/5402633/143723626-c360cce6-321e-4344-821d-b159aca11df2.png)

Reply:
I agree, result looks better with a unique winner. Only takes around 1s more on my machine. Thanks again for fine tuning!